src/e2eTests/java/uk/gov/hmcts/sptribs/e2e/enums/CaseParties.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.sptribs.e2e.enums;
+
+public enum CaseParties {
+    Subject("Subject"),
+    Applicant("Applicant"),
+    Representative("Representative");
```

Comment:
Is it good to add respondent and tribunals as case parties and add respective tests?

Reply:
fair enough